### PR TITLE
Feat/add creator dao stats to platform orchestrator

### PR DESCRIPTION
**Pull request by ravi-sawlani-yral**
## Changes
- added creator dao stats in platform orchestrator
- added supporting methods
- added methods to collect preexisting tokens

### CODE REVIEW

File: src/canister/platform_orchestrator/src/data_model/mod.rs [hunk 1]
```diff
@@ -131,3 +138,16 @@ impl Default for CanisterUpgradeStatus {
...
     }
 }
+
+impl CanisterData {
+    pub fn add_creator_dao_stats_recieved_from_subnet_orchestrator(
+        &mut self,
+        individual_user_profile_id: Principal,
+        root_canister_ids: HashSet<Principal>,
+    ) {
+        root_canister_ids.iter().for_each(|governance_canister_id| {
```
Comment (komal-sai-yral) [inline src/canister/platform_orchestrator/src/data_model/mod.rs:148]: naming issue : Isn't governance_canister_id different from root canister ?

File: src/canister/individual_user_template/src/api/cdao/send_creator_dao_stats_to_subnet_orchestrator.rs [hunk 2]
```diff
@@ -0,0 +1,31 @@
+use std::collections::HashSet;
+
+use candid::Principal;
+use ic_cdk_macros::update;
+use shared_utils::common::utils::permissions::is_caller_controller;
+
+use crate::{util::subnet_orchestrator::SubnetOrchestrator, CANISTER_DATA};
+
+#[update(guard = "is_caller_controller")]
```
Comment (komal-sai-yral) [inline src/canister/individual_user_template/src/api/cdao/send_creator_dao_stats_to_subnet_orchestrator.rs:9]: curious : Can't a update called be called from a get call ?

Comment (ravi-sawlani-yral) [reply src/canister/individual_user_template/src/api/cdao/send_creator_dao_stats_to_subnet_orchestrator.rs:9]: inter canister callls are update calls

File: src/lib/shared_utils/src/types/creator_dao_stats.rs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+struct IndividualUserCreatorDaoEntry {
+    individual_profile_id: IndividualProfileId,
+    deployed_canisters: HashSet<RootCanisterId>,
+}
+
+impl CreatorDaoTokenStats {
+    pub fn insert_new_entry(
+        &mut self,
+        individual_user_profile_id: IndividualProfileId,
+        governance_canister_id: RootCanisterId,
```
[the remark this answers is not in the document]
Comment (ravi-sawlani-yral) [reply src/lib/shared_utils/src/types/creator_dao_stats.rs]: fixed

File: src/canister/platform_orchestrator/src/data_model/mod.rs (hunk 1 above)
Comment (ravi-sawlani-yral) [reply src/canister/platform_orchestrator/src/data_model/mod.rs]: fixed.

**Review by komal-sai-yral — APPROVED** [review]

**Pull request merged**